### PR TITLE
CNV-11965: for defect 1959287 CPU vendors

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with [CNV-11965](https://issues.redhat.com/browse/CNV-11965) and https://bugzilla.redhat.com/show_bug.cgi?id=1959287 for the CNV 4.8 release.

### CODE REVIEW

**Comment by lmandavi** [comment]
@fabiand @ksimon1 - Please review and provide your input. Thanks!
https://deploy-preview-34300--osdocs.netlify.app/openshift-enterprise/latest/virt/install/preparing-cluster-for-virt

File: virt/install/preparing-cluster-for-virt.adoc [hunk 1]
```diff
@@ -41,6 +41,12 @@ Without an external monitoring system or a qualified human monitoring node healt
 
 * If your cluster uses worker nodes from multiple CPU vendors, live migration failures can occur. For example, a virtual machine with an AMD CPU might attempt to live-migrate to a node with an Intel CPU and likely fail migration. To avoid this, label nodes with a vendor-specific label, such as `Vendor=Intel` or `Vendor=AMD`, and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information.
 
+* All CPUs must support the following requirements:
+
```
Comment (fabiand) [inline virt/install/preparing-cluster-for-virt.adoc:45]: Please also add: "Supported by RHEL 8"

**Review by fabiand — CHANGES_REQUESTED** [review]
One requirement was missing

File: virt/install/preparing-cluster-for-virt.adoc (hunk 1 above)
Comment (lmandavi) [reply virt/install/preparing-cluster-for-virt.adoc:45]: Added the RHEL 8 requirement.

**Comment by lmandavi** [comment]
@fabiand - Please review. 
https://deploy-preview-34300--osdocs.netlify.app/openshift-enterprise/latest/virt/install/preparing-cluster-for-virt

File: virt/install/preparing-cluster-for-virt.adoc [hunk 2]
```diff
@@ -41,6 +41,13 @@ Without an external monitoring system or a qualified human monitoring node healt
...
+** Red Hat Enterprise Linux 8
```
Comment (fabiand) [inline virt/install/preparing-cluster-for-virt.adoc:46]: This is technically a little inprecise, because not the CPU is supporting RHEL 8, but RHEL 8 is supporting the CPU.
But it might be good enough

**Review by fabiand — COMMENTED** [review]
A nit

/lgtm

File: virt/install/preparing-cluster-for-virt.adoc (hunk 2 above)
Comment (lmandavi) [reply virt/install/preparing-cluster-for-virt.adoc:46]: Let me redo the text for clarity, thanks for the clarification!

**Comment by lmandavi** [comment]
Updated the content with latest input from @fabiand. @ipinto Requesting QE input.

**Comment by lmandavi** [comment]
QE approval received in the Jira story. Requesting peer review.

File: virt/install/preparing-cluster-for-virt.adoc [hunk 3]
```diff
@@ -41,6 +41,12 @@ Without an external monitoring system or a qualified human monitoring node healt
 
 * If your cluster uses worker nodes from multiple CPU vendors, live migration failures can occur. For example, a virtual machine with an AMD CPU might attempt to live-migrate to a node with an Intel CPU and likely fail migration. To avoid this, label nodes with a vendor-specific label, such as `Vendor=Intel` or `Vendor=AMD`, and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information.
 
+* All CPUs must have Red Hat Enterprise Linux 8 and support the following requirements:
```
Comment (sjhala-ccs) [inline virt/install/preparing-cluster-for-virt.adoc:44]: Suggest rewriting this for clarity - "All CPUs must be supported by RHEL 8 and meet the following requirements:"

File: virt/install/preparing-cluster-for-virt.adoc [hunk 4]
```diff
@@ -41,6 +41,12 @@ Without an external monitoring system or a qualified human monitoring node healt
...
+
+** Intel 64 or AMD64 CPU extensions
```
Comment (sjhala-ccs) [inline virt/install/preparing-cluster-for-virt.adoc:46]: Suggest rewording the list items for parallel construction

**Review by sjhala-ccs — COMMENTED** [review]
A couple of suggestions, otherwise LGTM

File: virt/install/preparing-cluster-for-virt.adoc (hunk 3 above)
Comment (lmandavi) [reply virt/install/preparing-cluster-for-virt.adoc:44]: Earlier, I had text which mentioned what you suggested "All CPUs must be supported by RHEL 8..." and Fabian noted the following in this PR earlier:

Fabian's note:
This is technically a little inprecise, because not the CPU is supporting RHEL 8, but RHEL 8 is supporting the CPU.
But it might be good enough

@lmandavi lmandavi 21 hours ago Author Contributor
Let me redo the text for clarity, thanks for the clarification!

@sjhala-ccs - so let me know your thoughts. Can we leave the sentence as it is currently stated. If not, how do you recommend we reposition it?

Comment (sjhala-ccs) [reply virt/install/preparing-cluster-for-virt.adoc:44]: @lmandavi As far as I understand my suggested rewrite aligns with what Fabian said earlier that the CPU is not supporting RHEL 8, but RHEL 8 is supporting the CPU.

The sentence "All CPUs must be supported by RHEL 8 and meet the following requirements" conveys that aspect more clearly than the current version IMO.

File: virt/install/preparing-cluster-for-virt.adoc (hunk 4 above)
Comment (lmandavi) [reply virt/install/preparing-cluster-for-virt.adoc:46]: Reworded.

**Review by ousleyp — APPROVED** [review]
